build-system/runner/src/org/ampproject/AmpPass.java

```diff
@@ -100,26 +99,36 @@ public AmpPass(AbstractCompiler compiler, boolean isProd,
    * and a child STRING "assert" (or any other signature from Set<String> value)
    */
   private boolean isCallRemovable(Node n) {
-    if (n != null && n.isCall()) {
-      Node grandparent = n.getGrandparent();
-      // We want to make sure an `Log.prototype` method is actually invoked and not just
-      // a pointer to it.
-      if (grandparent == null || !grandparent.isCall()) {
-        return false;
-      }
-      Node callname = n.getFirstChild();
-      Node getprop = n.getNext();
-      if (callname != null && callname.isGetProp()) {
-       Set<String> methodCallNames = stripTypeSuffixes
-          .get(callname.getQualifiedName());
-       if (methodCallNames != null) {
-         for (String methodCallName : methodCallNames) {
-           if (getprop != null && getprop.isString() &&
-               methodCallName == getprop.getString()) {
-             return true;
-           }
-         }
-       }
+    if (n == null || !n.isCall()) {
```

Comment:
thanks, is there any action item you want me. (besides the possible 2 phase pass refactor)